community/milind/Zarr_file_example/Zarr_file_example.py

```diff
@@ -0,0 +1,45 @@
+@fused.udf
+def udf(bbox: fused.types.Bbox = None, layer: str = "lccs_class_7"):
+    import math
+    import os
+
+    import geopandas as gpd
+    import numpy as np
+    import rioxarray
+    import shapely
+    import xarray as xr
+
+    utils = fused.load(
+        "https://github.com/fusedio/udfs/tree/f928ee1/public/common/"
+    ).utils
+
+    ds = xr.open_zarr("s3://fused-users/fused/plinio/seasfire_v3/")
+
+    print("bbox", bbox)
```

Comment:
`bbox` needn't be printed. 
On the other hand, printing `ds` is useful to the user.


```suggestion
    print("bbox", bbox)
    
```

```suggestion
```